### ISSUE TITLE
Airbrake failing with undefined method `notify_request’ as of 3.2

### ISSUE THREAD

**Issue by brchristian** [airbrake/airbrake-ruby#406]
(Note: duplicate of https://github.com/airbrake/airbrake/issues/895 in case this is the more appropriate place for this issue.)

As of updating airbrake-ruby 3.2 today, every pageview on our site throws a 500 error:

```
undefined method `notify_request' for #<Airbrake::NoticeNotifier:0x00007ff0ca266520> Did you mean? notify_sync
```
The exception is being thrown from https://github.com/airbrake/airbrake/blob/master/lib/airbrake/rails/action_controller_subscriber.rb#L20

Seems like something is going wrong here if airbrake and airbrake-ruby don’t play nicely together!

We are rolling back to airbrake-ruby 3.1 in the meantime...

**Comment by kyrylo** [comment]
Yeah, there were substantial changes in 3.2 and the airbrake hasn't been updated yet. Sorry about that, I didn't realise it would cause a breakage. I released 3.2 today and I plan to cut a new release of the airbrake gem on Monday.

**Comment by kyrylo** [comment]
Please update to v3.2.2.
